bundle-workflow/src/run_integ_test.py

```diff
@@ -37,6 +38,9 @@ def parse_arguments():
     parser.add_argument(
         "--test-manifest", type=argparse.FileType("r"), help="Test Manifest file."
     )
+    parser.add_argument(
+        "--test-run-id", type=str, help="Test Run ID"
```

Comment:
You might want to sync up with @setiah as he also added this param as part of #456